### DISCUSSION TITLE
JIT, plugins and purge

### DISCUSSION THREAD

**Discussion by pascalmann** [tailwindlabs/tailwindcss#4014] [Help]
Hi,

Why these different bunddle sizes in production build ?
I made some tests with or without ```mode: 'jit```, and with our without plugins (forms, typography, line-clamp, aspect-ratio).

### Minimal Nx project with just some classes (text-center sm:text-left) in one template file : 
**with jit**
styles.904bf782f5f9158df4aa.css   | styles        |   **2.98 kB**

**with jit and with plugins**
styles.dad1e55e78a0ec8f956c.css   | styles        |   **7.27 kB**

**without jit**
styles.e7b26c7ecfe6961a07c8.css   | styles        |   **2.86 kB**

**without jit and with plugins**
styles.93390850b63a952d6f35.css   | styles        |   **4.32 kB**

### Bigger Nx project but also with just some classes (text-center sm:text-left) in only one template file : 
**without jit and with plugins**
styles.4d145e03749800d9aefe.css          | styles               |  **156.76 kB**

**with jit and with plugins**
styles.c8d65b8d4dc0581aa23f.css          | styles               |  **330.64 kB**

**Comment by binarious** [comment]
I've seen a similar thing. A smaller project:
**without jit**
`dist/assets/main.d720b8a8.css    17.82 KiB / gzip: 3.72 KiB`

**with jit**
`dist/assets/main.7fcd448c.css    28.34 KiB / gzip: 5.12 KiB`

I'm currently disabling JIT for production builds, because of this.
